### PR TITLE
Refactor: fix mismatch of Scope class member declaration

### PR DESCRIPTION
**Pull request by 9rnsr**
Separated from #5750.

### CODE REVIEW

**Comment by mathias-lang-sociomantic** [comment]
Why the change of style ? Otherwise :+1:

**Comment by 9rnsr** [comment]
Without the style change, the bugfix would break vertical alignment of line comments in `scope.h`. I think placing comment in own line would be better to reduce bike-shed discussion.

**Comment by mathias-lang-sociomantic** [comment]
> Without the style change, the bugfix would break vertical alignment of line comments in scope.h. I think placing comment in own line would be better to reduce bike-shed discussion.

Fine by me (I tend to prefer this style anyway).
Maybe we could have a style guide (as part of a `CONTRIBUTING.md` file) to reduce the amount of bike shedding even more ?

**Comment by 9rnsr** [comment]
Yes... I think we should do it as fast as possible.

Walter is not active for such the *guideline* documents, then someone should work for that.

To me writing long documentation is much painful than contributing code, because I'm not a native English user. But I would have to do it...

**Comment by mathias-lang-sociomantic** [comment]
I'll gladly help if you can provide the key point.
Could you contact me via email ? Both my personal and company email are on bugzilla under my real name.

**Comment by 9rnsr** [comment]
Thanks for your offer, but the guideline would be a list of *my* points in the first time. If it would not fit the Walter's points while reviewing, your work would be wasted. So first I should write it in my responsibility.
After the PR opened, I'll welcome review comments.

File: src/dscope.d [hunk 1]
```diff
@@ -155,7 +155,9 @@ struct Scope
     int explicitProtection;         // set if in an explicit protection attribute
 
     StorageClass stc;               // storage class
-    DeprecatedDeclaration depdecl;  // customized deprecation message
+
+    // customized deprecation message
+    DeprecatedDeclaration depdecl;
```
Comment (WalterBright) [inline src/dscope.d]: I see no reason for this change. It's just churn.

File: src/scope.h [hunk 2]
```diff
@@ -113,7 +113,9 @@ struct Scope
     int explicitProtection;     // set if in an explicit protection attribute
 
     StorageClass stc;           // storage class
-    char *depmsg;               // customized deprecation message
+
+    // customized deprecation message
+    DeprecatedDeclaration *depdecl;
```
Comment (WalterBright) [inline src/scope.h]: Please use one line, not 3. We're not introducing a new style for how fields are commented.

Comment (9rnsr) [inline src/scope.h]: There's not yet proper style guide to be followed by us. So your argument is still a personal opinion.
I have a free not to follow your comment, and you have a free not to fix the bug.

(And there's a third way - you can open another PR with the style you favor.)

Comment (WalterBright) [inline src/scope.h]: You are **changing** the existing style. This is not going to happen. (You're right it's a personal preference - and there are 123 contributors each with their own preferences.)

**Comment by yebblies** [comment]
> I think placing comment in own line would be better to reduce bike-shed discussion.

It seems you're wrong about this.

**Comment by WalterBright** [comment]
Auto-merge toggled on

**Pull request merged**